src/web/components/Core/PortalHeader.tsx

```diff
@@ -68,7 +72,21 @@ export function PortalHeader({ email, fullname, setDarkMode = undefined }: Porta
             </div>
           </DropdownMenuItem>
           <DropdownMenuSeparator className='separator' />
-          <DropdownMenuItem>Log out</DropdownMenuItem>
+          <DropdownMenuItem onSelect={(event) => event.preventDefault()}>
```

Comment:
It looks like you have a merge error here - you've got a duplicated dark-mode switch.